libiocage/lib/Jail.py

```diff
@@ -18,6 +18,49 @@
 
 
 class Jail:
+    """
+    Iocage unit orchestrates a jail's configuration and manages state
+
+    Jails are represented as zfs dataset `zpool/iocage/jails/<NAME>`
+
+    Directory Structure:
+
+        zpool/iocage/jails/<NAME>: The jail's dataset containing it's
+            configuration and root directory. Iocage legacy used to store
+            a jails configuration as ZFS properties on this dataset. Even
+            though the modern JSON config mechanism is preferred.
+
+        zpool/iocage/jails/<NAME>/root: This directory is the dataset
```

Comment:
This can just be said as `This dataset is used as the jail's root when starting the jail`. Perhaps instead of origin we say `a clone's root or release's root dataset are normally the sources`